### DISCUSSION TITLE
Fit images inside a table cell

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#530] [Q&A] [answered]
Dear @pgundlach 

Is there a method to fit an image inside a cell to a table with fixed width?

Thanks! 🙏

**Comment by pgundlach** [comment, marked as answer]
No there is no way for that. You need to know the exact dimensions.

**Reply by Cicorione** [reply]
Would it be the same if the image is the cell's background?

**Reply by pgundlach** [reply]
No, there is no real background image.

**Reply by pgundlach** [reply]
(it is the same issue - so the answer should be 'yes')

**Reply by Cicorione** [reply]
Thanks!
